frontend/src/pages/Dashboard.jsx

```diff
@@ -1,14 +1,24 @@
+import { useEffect, useState } from "react";
 import DashboardCard from "../components/DecisionCard";
 
 function Dashboard() {
+  const [decisions, setDecisions] = useState([]);
+  useEffect(() => {
+    fetch("http://localhost:5000/decisions-cards")
+      .then((res) => res.json())
+      .then((data) => setDecisions(data));
+  }, []);
   return (
     <section>
       <h1>Décisions en cours</h1>
       <div className="running-decision-dashboard">
-        <DashboardCard />
-        <DashboardCard />
-        <DashboardCard />
-        <DashboardCard />
+        {decisions.map((decision) => (
+          <DashboardCard
+            key={decision.id}
```

Comment:
the decision id is repeated if there are several users attached to the decision so your key won't be unique